src/manifest.json

```diff
@@ -1,8 +1,12 @@
 {
-
   "description": "An add-on to let you snooze your tabs for a while.",
   "manifest_version": 2,
   "name": "snoozetabs",
+  "author": "Blake Winton <bwinton@latte.ca>",
```

Comment:
For TabCenter, I've used an author of [Mozilla](https://github.com/bwinton/TabCenter/blob/master/package.json#L103), since I'm really writing it on their dime.  And listed myself as a contributor.